src/domain/profile/Profile.tsx

```diff
@@ -28,19 +29,26 @@ const Profile: FunctionComponent = () => {
     if (profile) dispatch(saveProfile(profile));
   }
   return (
-    <Container className={styles.grayBackground}>
-      <div className={styles.profileWrapper}>
-        <h1>{t('profile.heading')}</h1>
-        <div className={styles.childInfo}>
-          {profile &&
-            profile.children.map(child => (
-              <p key={child.id}>{child.firstName}</p>
-            ))}
-        </div>
+    <HelmetProvider>
```

Comment:
Wrap the Layout with the Provider, not each of page

Reply:
That requires me to put the <Helmet> inside the <Container> on the Home page for example - are you ok with that?